data/txHandler.go

```diff
@@ -147,6 +146,21 @@ func (handler *TxHandler) backlogWorker() {
 			if handler.checkAlreadyCommitted(wi) {
 				continue
 			}
+
+			// build the transaction verification context
+			latest := handler.ledger.Latest()
+			latestHdr, err := handler.ledger.BlockHdr(latest)
+			if err != nil {
+				logging.Base().Warnf("Could not get header for previous block %v: %v", latest, err)
+				continue
+			}
```

Comment:
This code is the same at both callers of `prepareVerifyContexts`, so perhaps it should go inside `prepareVerifyContexts`

Reply:
I'll leave out fetching the latest header in the factored code, since the ledger evaluator does not fetch the latest header; instead, it uses the header of the block it is currently building/validating.